nmdc_server/schemas_submission.py

```diff
@@ -47,6 +47,7 @@ class NmcdAddress(BaseModel):
     city: str
     state: str
     postalCode: str
+    country: str
```

Comment:
```suggestion
    country: str = ""
```
I came across a `500` status code when loading the submission landing page. Looks like there's some backend validation that is not happy.

Since `country` is a new required field, we need to stay backwards-compatible with existing submissions. There are a couple of options.

1. We could have the serializer add a default value (empty string?)
2. We could write a migration to add a default value

I don't mind sticking with option 1, but we also need to make sure that any consumers of this data are aware that `country` might be missing (I'm not sure how robust the submission to mongo pipeline is about this).

I also don't know what the default value should be. The empty string allows us to sneak existing submissions past the pydantic classes, but these submissions are still technically invalid. It's probably fine though.

Reply:
fixed in d30368d5c437c4425ac72c382d057e9815fb4f22
As discussed with team, the default was set to `USA`